app/.node-version

```diff
@@ -0,0 +1 @@
+16
```

Comment:
Can we use the latest LTS version? I think we're at 18 now.

Reply:
This is recommended by the React Native team, but I can try with 18 from here on out. If nothing goes wrong I'll change this in future PRs.